new_arrivals_chi/app/templates/base.html

```diff
@@ -43,8 +43,14 @@
 </body>
 
 <form action="/" method="GET">
-    <button type="submit" class="language-button">{{ translations[language]['util']['language'] | escape }}</button>
-    <input type="hidden" name="lang" value="{{ 'es' if language == 'en' else 'es' | escape }}">
+    <button type="submit" class="language-button">
+        {% if language == 'es' %}
+            English
+        {% else %}
+            Español
+        {% endif %}
```

Comment:
I'll go back to the JSON structure on my PR

Reply:
Ty!